app/api.py

```diff
@@ -580,6 +625,60 @@ def channel_acl(self, id, channel_id):
         data = obj_to_dict(server.getACL(channel_id))
         return Response(json.dumps(data, sort_keys=True, indent=4), mimetype='application/json')
 
+    @conditional(auth.login_required, auth_enabled)
+    @route('<int:id>/channels/<int:channel_id>/acl', methods=['PATCH'])
+    def update_channel_acl(self, id, channel_id):
+        """ Update specific channel ACL
+        """
+
+        server = meta.getServer(id)
+
+        # Return 404 if not found
+        if server is None:
+            return jsonify(message="Not Found"), 404
+
+        origin_acl = server.getACL(channel_id)
+        update_acls = origin_acl[0]
+        update_groups = origin_acl[1]
+        update_inherit = origin_acl[2]
```

Comment:
Can you explain what's going on here with the ACL settings. It's been a while since I've worked on Murmur Slice API, so I'm a bit rusty. :)

Reply:
oh this part, I want to follow RESTful PATCH.
Example :
If this request only pass `acls`, not pass `groups` and `inherit`, will only update acls.
update_groups will be groups of `origin_acl`, update_inherit will be inherit of `origin_acl`.
And update_acls will be `new_acls`

on `server.setACL(channel_id, update_acls, update_groups, update_inherit)`